src/locales/en.json

```diff
@@ -8,12 +8,16 @@
     "logoutBtn": "Logout"
   },
   "signup": {
-    "title": "Sign up",
+    "title": "SIGN UP",
```

Comment:
you can keep lowercases and use text transform with css.

Reply:
Will do!